src/outputs/output_parameters.hpp

```diff
@@ -0,0 +1,58 @@
+//========================================================================================
```

Comment:
Including `outputs/outputs.hpp` everywhere I needed it to make `OutputType` an argument to the `UserWork` callbacks introduced a nasty circular dependency. So now only this file, which doesn't contain any mesh or pack types, is included elsewhere.